mmaction/models/backbones/st_gcn.py

```diff
@@ -0,0 +1,449 @@
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from ..builder import BACKBONES
+
+
+def zero(x):
+    return 0
+
+
+def iden(x):
+    return x
+
+
+def get_hop_distance(num_node, edge, max_hop=1):
+    A = np.zeros((num_node, num_node))
```

Comment:
1. for common variables rather than constant, we use lower case
2. `A` is not a good name for its ambiguous